addons/account/views/account_move_views.xml

```diff
@@ -537,6 +537,10 @@
                          attrs="{'invisible': ['|', '|', ('move_type', 'not in', ('in_invoice', 'in_refund')), ('invoice_has_outstanding', '=', False), ('payment_state', 'not in', ('not_paid', 'partial'))]}">
                         You have <bold><a class="alert-link" href="#outstanding" role="button">outstanding debits</a></bold> for this supplier. You can allocate them to mark this bill as paid.
                     </div>
+                    <!-- Vendor bill currency consistency -->
+                    <div class="alert alert-warning" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('is_inactive_currency', '=', False)]}">
+                        In order to validate this invoice, you must activate the <button class="oe_link" type="object" name="action_open_currency" style="padding: 0; vertical-align: baseline;">currency</button> of the bill. The journal entries need to be computed by Odoo before being posted in your company's currency.
```

Comment:
lot of in-line styling in there. Is everything really needed?

Reply:
I based myself on the styling of the other warnings above to stay consistent with them.

Reply:
might be worth creating/modifying the css of the class instead of copying/pasting it several times

Reply:
or use class mb0 is the same than your in-line style.

Reply:
Hi @nhomar , this is exactly what I just did :)